src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_ProfileStack/Profile_RootScreen.tsx

```diff
@@ -371,141 +265,45 @@ export const Profile_RootScreen = ({navigation}: ProfileProps) => {
-                        .inspection.label,
-                    )}
-                    testID="inspectionInfoButton"
-                    onPress={() => Linking.openURL(inspectionInfoUrl)}
-                    accessibility={{
-                      accessibilityHint: t(
-                        ProfileTexts.sections.information.linkSectionItems
-                          .inspection.a11yLabel,
-                      ),
-                      accessibilityRole: 'link',
-                    }}
-                  />
-                )}
-
-                {refundInfoUrl && (
-                  <LinkSectionItem
-                    icon={<ThemeIcon svg={ExternalLink} />}
-                    text={t(
-                      ProfileTexts.sections.information.linkSectionItems.refund
-                        .label,
-                    )}
-                    testID="refundInfoButton"
-                    onPress={() => Linking.openURL(refundInfoUrl)}
-                    accessibility={{
-                      accessibilityHint: t(
-                        ProfileTexts.sections.information.linkSectionItems
-                          .refund.a11yLabel,
-                      ),
-                      accessibilityRole: 'link',
-                    }}
-                  />
-                )}
-                {a11yStatementUrl && (
-                  <LinkSectionItem
-                    icon={<ThemeIcon svg={ExternalLink} />}
-                    text={t(
-                      ProfileTexts.sections.information.linkSectionItems
-                        .accessibilityStatement.label,
-                    )}
-                    testID="a11yStatementButton"
-                    onPress={() => Linking.openURL(a11yStatementUrl)}
-                    accessibility={{
-                      accessibilityHint: t(
-                        ProfileTexts.sections.information.linkSectionItems
-                          .accessibilityStatement.a11yLabel,
-                      ),
-                      accessibilityRole: 'link',
-                    }}
-                  />
-                )}
+                <LinkSectionItem
+                  text={t(ProfileTexts.sections.information.label)}
+                  onPress={() =>
+                    navigation.navigate('Profile_InformationScreen')
+                  }
+                  testID="informationButton"
+                />
               </Section>
             </>
           )}
+          <ContentHeading text={t(ProfileTexts.sections.contact.heading)} />
+          <Section>
+            <LinkSectionItem
+              text={t(ProfileTexts.sections.contact.helpAndContact)}
+              onPress={() => {}}
+              testID="contactHelpButton"
+            />
+            <LinkSectionItem
+              text={t(ProfileTexts.sections.contact.feedback)}
+              onPress={() => {}}
+              testID="contactFeedbackButton"
+            />
```

Comment:
I think we should leave these out if they're due to be added in another issue. Just so that master branch is release ready.